lib/colorls/core.rb

```diff
@@ -254,14 +259,30 @@ def group_info(group)
     end
 
     def size_info(filesize)
-      size = Filesize.new(filesize).pretty.split
-      size = "#{size[0][0..-4].rjust(4,' ')} #{size[1].ljust(3,' ')}"
+      filesize = Filesize.new(filesize)
+      size = @show_human_readable_size ? filesize.pretty : filesize.to_s
+      size = size.split
+      size = "#{size[0][0..-4].rjust(chars_for_size,' ')} #{size[1].ljust(3,' ')}"
```

Comment:
Currently this has too much space between the size and the date column when using non-human-readable sizes as it always left justifies the "B" within 3 spaces.

Since it will always be a single character in size, the ljust call should be removed.